src/lib/modules/PresentationModule.ts

```diff
@@ -0,0 +1,24 @@
+import { ConnectionRecord } from '../storage/ConnectionRecord';
+import { createOutboundMessage } from '../protocols/helpers';
+import { MessageSender } from '../agent/MessageSender';
+import { presenationRequestTemplate, PresentationService } from '../protocols/presentproof/PresentationService';
+
+export class CredentialsModule {
```

Comment:
Don't forget to rename `CredentialsModule`. I'm thinking if the name `Proof` wouldn't be better. So `ProofsModule`, `ProofService`, `ProofRecord`, ... I looked at .NET and they also use this name.

Reply:
Sure, will update that. Thanks for your input. I will update the test as well accordingly